ddl/table.go

```diff
@@ -748,7 +748,8 @@ func onUpdateFlashReplicaStatus(t *meta.Meta, job *model.Job) (ver int64, _ erro
 	}
 	if tblInfo.TiFlashReplica == nil || (tblInfo.ID == physicalID && tblInfo.TiFlashReplica.Available == available) ||
 		(tblInfo.ID != physicalID && available == tblInfo.TiFlashReplica.IsPartitionAvailable(physicalID)) {
```

Comment:
I think it will return nil in `UpdateTableReplicaInfo` in `ddl_api.go` rather than return error in `onUpdateFlashReplicaStatus` at `ddl_woker.go`. 
you mean for the jobs already in queue? do the double check?
Then, should it return error in `UpdateTableReplicaInfo` in `ddl_api.go`?